test/writer_test.py

```diff
@@ -365,6 +365,13 @@ def test_writer_copy_and_update():
         writer.copc_config.copc_extents.has_extended_stats
         == orig.copc_config.copc_extents.has_extended_stats
     )
+    # Check that other attributes have been copied
+    assert writer.copc_config.copc_info.spacing == orig.copc_config.copc_info.spacing
+    assert writer.copc_config.las_header.guid == orig.copc_config.las_header.guid
+    assert (
+        writer.copc_config.copc_extents.intensity.minimum
```

Comment:
u may want to change this from minimum to maximum (or a diff attribute) since the minumum will always be 0